scripts/release.sh

```diff
@@ -1,11 +1,14 @@
+#! /bin/bash
+
 # Usage: sh release.sh
 # Note: To run this script locally you need to export environment variables CIRCLE_TAG and GITHUB_TOKEN.
-#!/bin/bash
+
 FUTURE_RELEASE_SHA=$(hub rev-parse HEAD)
 LATEST_RELEASE=$(hub release)
 LATEST_RELEASE_SHA=$(hub rev-parse ${LATEST_RELEASE})
 LATEST_RELEASE_NEXT_COMMIT_SHA=$(hub log ${LATEST_RELEASE}..HEAD --oneline --pretty=%H | tail -n1)
-
+mkdir -p ./build/_output/docs/
+mkdir -p ./build/_output/bin/
```

Comment:
this is already created by go build right?

Reply:
Yeap, you are right worked without this line too. Thanks for pointing.